sdk/typescript/src/types/transactions.ts

```diff
@@ -133,7 +133,7 @@ export type SuiJsonValue =
 
 export type EmptySignInfo = object;
 export type AuthorityName = string;
-export type AuthoritySignature = string;
+export type AuthoritySignature = object;
```

Comment:
cc @666lcz

Reply:
This could actually break the Explorer: https://explorer.devnet.sui.io/transactions/6H22oWAgSyzdRKhzsjtL16uH3zSiGB%2BC5V5kdqa4OvY%3D
![CleanShot 2022-08-15 at 22 34 37](https://user-images.githubusercontent.com/76067158/184805220-2fd0fc6a-c961-4279-8819-0e7280540be8.png)


Right now we are displaying the validator signatures. Not sure what it would look like with this change. Is it possible to do some conversion to string at the RPC server, i.e., convert {R_bytes, s_bytes} to strings like what's in the screenshot?

Reply:
Transaction signature format seems to be evolving pretty quickly. I believe Sui supports multiple types of sigs and aggregated signatures. We may want to include the sig type in the explorer. @joyqvq may be you can sync up with @666lcz on the best way to display signatures in the explorer?